src/charts/LeafletMap.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React from 'react';
+import { GeoJsonObject } from 'geojson';
+import 'leaflet/dist/leaflet.css';
+import { LeafletMapProps } from '../interfaces/LeafletMap';
+import { MapContainer, TileLayer, GeoJSON } from 'react-leaflet';
+import Fullscreen from 'react-leaflet-fullscreen-plugin';
+
+const myStyle = {
```

Comment:
Rename 'myStyle' to something more specific like 'geoJsonStyle' or something else that is clearer?